app/components/PaymentModal.tsx

```diff
@@ -142,6 +162,21 @@ const PaymentForm: FC<{
           onChange={handleInputChange}
           data-cy="signer-account-input"
         />
+
+        <div>
+          <span data-cy="sign-transaction-message">Or sign with:</span>
+          {wallets.map((wallet, index) => {
+            if (wallet.getName() !== "secretKey")
```

Comment:
Can't we do something like `if (wallet.getName() !== SecretKey.NAME)`?

Reply:
I have imported the SecretKey class and made the suggested comparison to make the code clearer.